2-mandatory/1-weather-report.js

```diff
@@ -11,53 +11,83 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
+let cities = [
+  "London",
+  "Paris",
+  "Barcelona",
+  "Dubai",
+  "Mumbai",
+  "São Paulo",
+  "Lagos",
+];
+let citiesTemp = [
+  "London",
+  10,
+  "Paris",
+  12,
+  "Barcelona",
+  17,
+  "Dubai",
+  27,
+  "Mumbai",
+  29,
+  "São Paulo",
+  23,
+  "Lagos",
+  33,
+];
+
+function getTempratur(city) {
+  let index = citiesTemp.indexOf(city);
+  index++;
+  return citiesTemp[index];
 }
 
+function getTemperatureReport(cities) {
+  let tempratureReport = [];
+  // TODO
+  for (let i = 0; i < cities.length; i++) {
```

Comment:
Because you don't need the index (`i`) for your output here, you could use a `for of` loop instead.

Reply:
I agree, thank you.